<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendRankingHeroClass</class>
 <widget class="QDialog" name="SendRankingHeroClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>94</red>
        <green>47</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>94</red>
        <green>47</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>envoi Ã  RankingHero</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pokersphere.qrc">
    <normaloff>:/img/Resources/sphere.ico</normaloff>:/img/Resources/sphere.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="classementLabel">
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;a&quot; style=&quot;text-decoration:none;&quot;&gt;&lt;font color=&quot;#5E2F6A&quot;&gt;Classement :&lt;/font&gt;&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="labelVerticalLayout">
     <property name="leftMargin">
      <number>50</number>
     </property>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="annulerLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;a&quot; style=&quot;text-decoration:none;&quot;&gt;&lt;font color=&quot;#5E2F6A&quot;&gt;Annuler&lt;/font&gt;&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="envoyerLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;a&quot; style=&quot;text-decoration:none;&quot;&gt;&lt;font color=&quot;#5E2F6A&quot;&gt;Envoyer&lt;/font&gt;&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pokersphere.qrc"/>
 </resources>
 <connections/>
</ui>
